modules/snooker/src/main/ts/ephox/snooker/resize/Recalculations.ts

```diff
@@ -44,6 +44,16 @@ const recalculateWidth = function (warehouse: Warehouse, widths: number[]): Cell
   });
 };
 
+const recalculateWidthForColumns = (warehouse: Warehouse, widths: number[]): CellWidthSpan[] => {
+  const groups = Warehouse.justColumns(warehouse);
+
+  return Arr.map(groups, (cell: SugarElement, index: number) => ({
+    element: cell,
+    width: widths[index],
+    colspan: 1
```

Comment:
This likely won't work... what if you're loading in an existing table that has `<col span="2">`?  The calculations would then be incorrect here, so this also likely needs some more test cases added.

Reply:
This should also probably be pulled from the element, not hard coded.

Reply:
I also just checked and I can't see any tests added for this new function, unless I'm missing something (eg nothing was added to `RecalculationsTest.ts`).